code/datums/ammo/bullet/tank.dm

```diff
@@ -0,0 +1,74 @@
+/*
+//======
+					Tank Ammo
+//======
+*/
+
+/datum/ammo/bullet/tank/flak
+	name = "flak autocannon bullet"
+	icon_state = "autocannon"
+	damage_falloff = 0
+	flags_ammo_behavior = AMMO_BALLISTIC
+	accurate_range_min = 4
+
+	accuracy = HIT_ACCURACY_TIER_8
+	scatter = 0
+	damage = 60
+	damage_var_high = PROJECTILE_VARIANCE_TIER_8
+	penetration = ARMOR_PENETRATION_TIER_6
+	accurate_range = 32
+	max_range = 32
+	shell_speed = AMMO_SPEED_TIER_6
+
+/datum/ammo/bullet/tank/flak/on_hit_mob(mob/M,obj/projectile/P)
+	burst(get_turf(M),P,damage_type, 2 , 3)
```

Comment:
This would be a great example of the 'explosion ammo' trait. Definitely out of scope here though

Reply:
i have no idea how turning this into a trait would work...

Reply:
i am not againt's this PR being more than just being splitting a file into multiple files...

Reply:
true.